Classes/PhoneMainView.m

```diff
@@ -677,15 +677,15 @@ - (void)playMessageSoundForCallID:(NSString *)callID {
 }
 
 - (void)displayIncomingCall:(LinphoneCall *)call {
-//	LinphoneCallLog *callLog = linphone_call_get_call_log(call);
-//	NSString *callId = [NSString stringWithUTF8String:linphone_call_log_get_call_id(callLog)];
+	LinphoneCallLog *callLog = linphone_call_get_call_log(call);
+	NSString *callId = [NSString stringWithUTF8String:linphone_call_log_get_call_id(callLog)];
 
 	if ([UIApplication sharedApplication].applicationState != UIApplicationStateBackground) {
 		LinphoneManager *lm = [LinphoneManager instance];
-		//BOOL callIDFromPush = [lm popPushCallID:callId];
+		BOOL callIDFromPush = [lm popPushCallID:callId];
```

Comment:
Not a crash , it was disabled  because i wanted an auto-answer functionality.  this was one way to do it.